cmd/db-manager/v1beta1/Dockerfile

```diff
@@ -2,7 +2,6 @@
 FROM golang:alpine AS build-env
 
 ARG TARGETARCH
-ENV GRPC_HEALTH_PROBE_VERSION v0.4.15
```

Comment:
@tenzen-y @gaocegege @johnugeorge I am wondering, do we still need HealthServicer for our Suggestions ?
https://github.com/kubeflow/katib/blob/master/cmd/suggestion/hyperopt/v1beta1/main.py#L30

Reply:
@andreyvelich I think we need HealthServer. What reasons do you think we need not HealthServer?

Reply:
Yeah, basically this prob triggers our HealthServer that we expose in our Suggestions/DB-manager: https://github.com/grpc/grpc/blob/master/doc/health-checking.md.